lib/CL/devices/almaif/XrtDevice.hh

```diff
@@ -22,19 +22,42 @@
    IN THE SOFTWARE.
 */
 
-#ifndef XrtDevice_H
-#define XrtDevice_H
+#ifndef XRTDEVICE_H
+#define XRTDEVICE_H
 
 #include "AlmaIFDevice.hh"
 
+class XrtDRAMRegion;
+
 class XrtDevice : public AlmaIFDevice {
```

Comment:
Needs a proper class comment. At least a file comment of what this is really abstracting.

Reply:
added comment there